src/encoded/schemas/annotation.json

```diff
@@ -265,8 +265,8 @@
             "length": "long",
             "open_on_load": true
         },
-        "annotation_type": {
-            "title": "Annotation type",
+        "annotation_subtype": {
+            "title": "Annotation subtype",
```

Comment:
Thanks! I didn’t think about the case where both were needed, just with a different property name. VSCode is really strict about JSON properties, and had marked this one as duplication error.